argo_workflows/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f96fd144-a3c0-4fed-adcc-53f11f80ec04",
+  "app_id": "argo-workflows",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor the health and performance of Argo Workflows",
+    "title": "Argo_Workflows",
```

Comment:
I would probably rename it to `Argo Workflows` without underscores, which should be propagated to the readme and the changelog too

Reply:
uff, I remember that I had to **switch** to `Argo_Workflows`, but I forget **why**. I think maybe `ddev create` doesn't accept that?

Reply:
Done!